index.tsx

```diff
@@ -510,6 +510,18 @@ class DraggableFlatList<T> extends React.Component<Props<T>, State> {
     this.flatlistRef.current._component.scrollToOffset({ offset })
   })
 
+  scrollToIndex = (index) => {
+    this.flatlistRef.current._component.scrollToIndex(index)
+  }
+
+  scrollToEnd = (params) => {
+		this.flatlistRef.current._component.scrollToEnd(params)
```

Comment:
fix indentation and add types

Reply:
I fixed indentation and added types. Is it ok?